src/webui/api/torrentscontroller.cpp

```diff
@@ -1057,6 +1060,10 @@ void TorrentsController::addAction()
         }
     }
 
+    const QString downloaderParam = params()[u"downloader"_s];
```

Comment:
This change should be mentioned in WebAPI Changelog.
WebAPI version should be bumped.